src/Avalonia.Diagnostics/Diagnostics/ViewModels/ControlDetailsViewModel.cs

```diff
@@ -46,7 +47,8 @@ public ControlDetailsViewModel(TreePageViewModel treePage, IAvaloniaObject avalo
 
             if (avaloniaObject is StyledElement styledElement)
             {
-                styledElement.Classes.CollectionChanged += OnClassesChanged;
+                _classesChangedListener = new ActionClassesChangedListener(OnClassesChanged);
```

Comment:
It's a minor thing really, but couldn't we implement `IClassesChangedListener` on `ControlDetailsViewModel` instead of creating an inner object?

Reply:
Yeah, no problem. I originally chose the delegating implementation because I thought `Changed` was a bit nondescript as a method name, but with an explicit interface implementation its purpose becomes obvious again. Alternatively I could change the name to somethings like `ClassesChanged`, which would be fine since it's an internal interface with only one other implementor.

While I'm touching these files anyway I also did some general cleanup, feel free to drop those commits if you think that doesn't belong in this PR.